client/extensions/woocommerce/app/settings/shipping/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import ShippingLabels from './shipping-labels';
+import ShippingOrigin from './shipping-origin';
+import ShippingPackageList from './shipping-package-list';
+import ShippingZoneList from './shipping-zone-list';
+
+class Shipping extends Component {
```

Comment:
For components that will not end up having internal state or life cycle methods, we can use `stateless components`.

https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#stateless-functional-components